Code/src/main/java/nl/utwente/viskell/ui/components/DisplayBlock.java

```diff
@@ -101,7 +100,7 @@ public Expression getLocalExpr() {
     
     @Override
     public void refreshAnchorTypes() {
-        this.inputAnchor.setFreshRequiredType(TypeScope.unique("any"), new TypeScope());        
+        this.inputAnchor.setFreshRequiredType(this.getPane().getEnvInstance().buildType("Show a => a"), new TypeScope());        
```

Comment:
Would it make sense to have DisplayBlock keep an instance of this type and keep making fresh copies of it? This invokes the ANTLR parser every time.